src/components/secondPassphrase/index.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { mount } from 'enzyme';
+import chaiEnzyme from 'chai-enzyme';
+import { SecondPassphrase } from './index';
+
+chai.use(chaiEnzyme());
+chai.use(sinonChai);
+
+describe('SecondPassphrase', () => {
+  let wrapper;
+
+  const normalAccount = {
+    isDelegate: false,
+    address: '16313739661670634666L',
+    balance: 1000e8,
+  };
+
+  const withSecondPassAccount = {
+    isDelegate: true,
+    address: '16313739661670634666L',
+    balance: 1000e8,
+    secondSignature: 'sample phrase',
+  };
+
+  it('renders one MenuItem component for a normal account', () => {
+    const props = {
```

Comment:
To prevent code duplication, `props` can be defined outside `it`s without `account`. Appropriate `account` can be set inside each `it`.